build/index.js

```diff
@@ -266,12 +267,40 @@ function logSuccess(command, timeLapsed) {
     logMain(message);
 }
 
+function checkVersions(metaManifest) {
+    const thisNPM = process.platform === "win32" ? "npm.cmd" : "npm";
+    let exitCode = 0;
+    const nodeMajorVersion = parseInt(process.version.slice(1, 3), 10);
+    if (nodeMajorVersion < metaManifest.requiredNodeMajorVersion) {
+        console.error(
+            `⚠️ Node.js v${metaManifest.requiredNodeMajorVersion} or later ` +
+                `required, you are on ${process.version}`
+        );
+        exitCode = 1;
+    }
+    const npmVersion = execSync(`${thisNPM} --version`, { encoding: 'utf-8' });
```

Comment:
Just curious, what would happen if there simply _is_ no `npm` on the system here? Probably unlikely, but worth testing quickly at least manually I guess.

Reply:
Added a try-catch block to take care of that. The other option would be using which, which is not platform-independent.